tsconfig.json

```diff
@@ -19,6 +19,9 @@
     },
     {
       "path": "./shared/tasks"
+    },
+    {
+      "path": "./shared/stats"
```

Comment:
I'm still not 100% sold on this being in "shared"

Only because it's not shared, it's just used once in a single place

What do you think?

Reply:
Are we only ever going to use the performance stuff in github actions? I was under the impression that we were going to use it elsewhere.

But if not, yup, makes total sense to move it. If we want to split out the github bits from the performance bit, we can take @romaricpascal's suggestion and pop it somewhere in `.github`?

Otherwise just pop the whole thing in the base directory?

Reply:
The way I imagined it we'd run the performance stuff:
- from Github actions, to comment on PR
- locally, if we want a glimpse before commiting

Not sure if that makes it "shared". As an other thought, "shared" may actually be more "internal" to widen its responsibilities without us having to worry about how shared a piece of code is.

The Github commenting itself will be shared for sure, though. Hence my previous suggestion of having it as part of `.github`. Alternatively, it may fit neatly into the `helpers` package (as `helpers/github.js` for example). It'd feel neater than having it in the base directory? Would that solve the "shared" aspect of it?

Reply:
@romaricpascal Only asking because I've got to find a home for several other performance packages

Helpers sounds good 😊